cmd/bootstrap/cmd.go

```diff
@@ -0,0 +1,289 @@
+			`--param Region=us-west-2 --param AvailabilityZonesCount=2 --param Tags=Key1=Value1`,
+		Args:   cobra.ExactArgs(1),
+		Hidden: true,
+		Run:    run,
+	}
+	cmd.Flags().StringArrayVar(&args.params, "param", []string{}, "List of parameters")
+	return cmd
+}
+
+var args struct {
+	params []string
+}
+
+func run(cmd *cobra.Command, arg []string) {
+	r := rosa.NewRuntime().WithAWS().WithOCM()
+	defer r.Cleanup()
+
+	orgID, _, err := r.OCMClient.GetCurrentOrganization()
+	if err != nil {
+		r.Reporter.Errorf(err.Error())
+		os.Exit(1)
+	}
+
+	parsedParams, parsedTags := parseParams(args.params)
+
+	// Extract the first non-`--param` argument to use as the template command
+	var templateCommand string
+	for _, arg := range arg {
+		if !strings.HasPrefix(arg, "--param") {
+			templateCommand = arg
+			break
+		}
+	}
+
+	templateFile := selectTemplate(templateCommand)
+	if templateFile == "" {
+		r.Reporter.Errorf("No suitable template found for the specified parameters")
+		os.Exit(1)
+	}
+
+	r.OCMClient.LogEvent("RosaQuickStartVpc",
+		map[string]string{
+			ocm.Account:      r.Creator.AccountID,
+			ocm.Organization: orgID,
+			"template":       templateFile,
+		},
+	)
+
+	err = createVPCStack(templateFile, parsedParams, parsedTags)
+	if err != nil {
+		r.Reporter.Errorf(err.Error())
+		os.Exit(1)
+	}
+
+	r.Reporter.Infof("VPC created")
+}
+
+// parseParams converts the list of parameter strings into a map and sets default values
+func parseParams(params []string) (map[string]string, map[string]string) {
+	result := map[string]string{
+		"Name":                   "rosa-quickstart-default",
+		"VpcCidr":                "10.0.0.0/16",
+		"Region":                 "us-west-2",
+		"AvailabilityZonesCount": "2",
+	}
+
+	// Set default tags from the getTags function
+	defaultTags := getTags()
+	userTags := map[string]string{}
+
+	for _, param := range params {
+		parts := strings.SplitN(param, "=", 2)
+		if len(parts) == 2 {
+			if parts[0] == "Tags" {
+				tagEntries := strings.Split(parts[1], ",")
+				for _, entry := range tagEntries {
+					tagParts := strings.SplitN(entry, "=", 2)
+					if len(tagParts) == 2 {
+						userTags[tagParts[0]] = tagParts[1]
+					}
+				}
+			} else {
+				result[parts[0]] = parts[1]
+			}
+		}
+	}
+
+	// Combine default tags and user tags
+	combinedTags := mergeTags(defaultTags, userTags)
+
+	return result, combinedTags
+}
+
+// mergeTags combines default tags and user tags, giving precedence to user tags in case of conflicts
+func mergeTags(defaultTags, userTags map[string]string) map[string]string {
+	combinedTags := make(map[string]string)
+	for k, v := range defaultTags {
+		combinedTags[k] = v
+	}
+	for k, v := range userTags {
+		combinedTags[k] = v
+	}
+	return combinedTags
+}
+
+// selectTemplate selects the appropriate template file based on the parameters
+func selectTemplate(command string) string {
+	templates := map[string]string{
+		"rosa-quickstart-default-vpc":     "cmd/bootstrap/templates/vpc.yaml",
```

Comment:
@den-rgb I'd suggest to make the template directory structure match the requested "scenario". So if I'm requesting `rosa-quickstart-default-vpc`, then the CloudFormation template must exist at `cmd/bootstrap/templates/rosa-quickstart-default-vpc/cloudformation.yaml`

Directory-per-"scenario" means this function can be generic e.g check that the resource exists in the specified location, and means that we can embed additional metadata in the directory in the future if we wish e.g we could have `cmd/boostrap/templates/rosa-quickstart-default-vpc/metadata.json` in the future that provides lots of readable information for a `rosa describe bootstrap` command.